tests/warn/i21190.scala

```diff
@@ -0,0 +1,23 @@
+
+//> using options -Werror
```

Comment:
`-Werror` isn't needed here, `warn` tests will fail if an error is reported in an unspecified location.

Reply:
It's a way to say "expect no warnings" without scanning the source visually or grepping.

Reply:
Ok, that sounds like a good practice actually. :sweat_smile: